src/DistributedMetadata/MetadataService.cpp

```diff
@@ -28,6 +28,7 @@ namespace
     const String DATA_RETENTION_KEY = "data_retention";
     const String LOG_ROLL_SIZE_KEY = "log_roll_size";
     const String LOG_ROLL_PERIOD_KEY = "log_roll_period";
+    const String ENABLE_COMPRESS_INTERNAL_TOPIC = "cluster_settings.streaming_storage.kafka.enable_compress_internal_topic";
```

Comment:
This setting shall be better applied to per metadata topic instead of a global system streaming storage configuration ?

```
<system_catalogs>
<name>__system_catalogs</name>
<compression>true</compression>
...
</system_catalogs>
```